common/src/main/java/com/microsoft/hydralab/common/management/device/impl/IOSTestDeviceManager.java

```diff
@@ -131,6 +131,7 @@ public boolean uninstallApp(DeviceInfo deviceInfo, String packageName, Logger lo
 
     @Override
     public void resetPackage(DeviceInfo deviceInfo, String packageName, Logger logger) {
+        uninstallApp(deviceInfo, packageName, logger);
         classLogger.info("Nothing Implemented for iOS in " + currentMethodName());
```

Comment:
remove this line?

Reply:
Rewrite this method. Will not verify the implementation of TestDeviceManager is iOS anymore.
![image](https://user-images.githubusercontent.com/26757995/226828372-5f8a9127-5ab0-4de5-a1eb-9f64e212a626.png)